docs/source/pages/tutorials/exclude_include_checks.rst

```diff
@@ -0,0 +1,175 @@
+.. Copyright (c) 2024 - 2024, Oracle and/or its affiliates. All rights reserved.
+.. Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+=====================================
+Exclude and include checks in Macaron
+=====================================
+
+.. note::
+
+    This feature is only available from Macaron 0.8.0
+
+This tutorial will show how you can configure Macaron to:
+
+* exclude checks that you don't want to run.
+* include only checks that you want to run.
+
+------------------------------
+Installation and Prerequisites
+------------------------------
+
+This tutorial requires a high-level understanding of checks in Macaron and how they depend on each other. Please see this :ref:`page <macaron-developer-guide>` for more information.
+
+Skip this section if you already know how to install Macaron.
+
+.. toggle::
+
+    Please follow the instructions :ref:`here <installation-guide>`. In summary, you need:
+
+        * Docker
+        * the ``run_macaron.sh``  script to run the Macaron image.
+
+    .. note:: At the moment, Docker alternatives (e.g. podman) are not supported.
+
+
+    You also need to provide Macaron with a GitHub token through the ``GITHUB_TOKEN``  environment variable.
+
+    To obtain a GitHub Token:
+
+    * Go to ``GitHub settings`` → ``Developer Settings`` (at the bottom of the left side pane) → ``Personal Access Tokens`` → ``Fine-grained personal access tokens`` → ``Generate new token``. Give your token a name and an expiry period.
+    * Under ``"Repository access"``, choosing ``"Public Repositories (read-only)"`` should be good enough in most cases.
+
+    Now you should be good to run Macaron. For more details, see the documentation :ref:`here <prepare-github-token>`.
+
+------------------
+Motivating example
+------------------
+
+Let's say you want to run the analysis on `the micronaut-projects/micronaut-core Github repository <https://github.com/micronaut-projects/micronaut-core>`_ at the tag ``4.3.10``.
+
+Normally, this is how you would run Macaron:
+
+.. code-block:: shell
+
+  ./run_macaron.sh analyze -purl pkg:/github.com/micronaut-projects/micronaut-core@v4.3.10 --skip-deps
+
+By default, Macaron will run all of its checks. However, there can be checks that are not applicable to ``micronaut-projects/micronaut-core``.
```

Comment:
I think this sentence is a high-level description of the general cases.

```
By default, Macaron will run all of its checks. However, there can be checks that do not apply to the project of interest.
```

Because of this, I prefer it to be mentioned at the beginning, rather than after the example has been introduced.

Reply:
I have moved it to the beginning of the tutorial - https://github.com/oracle/macaron/pull/667/commits/44541c9e72aa223985fa36c7b0cfb3cdf77cd3f5